src/pages/MemberConsole.js

```diff
@@ -1,198 +1,89 @@
-    const dars = await PendingCases.findDataRequestPendingCases();
-
-    this.setState(prev => {
-      prev.currentUser = currentUser;
-      prev.totalDulPendingVotes = duls.totalDulPendingVotes;
-      prev.electionsList.dul = duls.dul;
-      prev.totalAccessPendingVotes = dars.totalAccessPendingVotes;
-      // Filter vote-able elections. See https://broadinstitute.atlassian.net/browse/DUOS-789
-      // for more work related to ensuring closed elections aren't displayed here.
-      prev.electionsList.access = _.filter(dars.access, (e) => { return e.electionStatus !== 'Closed'; });
-      return prev;
-    });
-
-  }
-
-  openAccessReview = (referenceId) => async () => {
-    const pathStart = NavigationUtils.accessReviewPath();
-    this.props.history.push(`${pathStart}/${referenceId}`);
-  };
-
-  handleSearchDul = (query) => {
-    this.setState({ searchDulText: query });
-  };
-
-  handleSearchDar = (query) => {
-    this.setState({ searchDarText: query });
-  };
-
-  searchTable = (query) => (row) => {
-    if (query !== undefined) {
-      let text = JSON.stringify(row);
-      return text.toLowerCase().includes(query.toLowerCase());
-    }
-    return true;
-  };
-
-  render() {
-
-    const { searchDarText, showModal, darDetails, researcher } = this.state;
-    const pageCount = Math.ceil((this.state.electionsList.access.filter(this.searchTable(searchDarText)).length).toFixed(1) / (this.state.accessLimit));
-    const closeSummaryModal = () => this.setState({ showModal: false });
-    const openSummaryModal = async (dar) => {
-      let darDetails;
-      await DAR.describeDar(dar.referenceId).then((resp) => { darDetails = resp; });
-      let researcherPromise;
-      let datasets;
-      if (!isNil(darDetails)) {
-        darDetails.researchType = DataUseTranslation.generateResearchTypes(darDetails);
-        await User.getById(darDetails.researcherId).then((resp) => { researcherPromise = resp; });
-        await getDatasets(darDetails).then((resp) => { datasets = resp; });
+import { updateLists as updateListsInit } from '../libs/utils';
+import { tableHeaderTemplate, tableRowLoadingTemplate } from '../components/dar_table/DarTable';
+import DarTableSkeletonLoader from '../components/TableSkeletonLoader';
+import { isNil } from 'lodash/fp';
+import {consoleTypes} from "../components/dar_table/DarTableActions";
+
+export default function NewMemberConsole(props) {
+  const [electionList, setElectionList] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+  const [tableSize, setTableSize] = useState(10);
+  const [descendantOrder, setDescendantOrder] = useState(false);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        setIsLoading(true);
+        const pendingList = await DAR.getDataAccessManageV2();
```

Comment:
It appears that I see more records in the member_console vs the new_member_console (250 vs 50) but I think that's because the old console displays DACs that I do not have permissions to access/action. I believe the behavior I see now with your changes is what we expect.

Perhaps those old functions were not working as inteneded:
    const duls = await PendingCases.findConsentPendingCasesByUser(currentUser.dacUserId);

Reply:
Most likely it's because we're grouping DARs into collections whereas before we interacted with each DAR individually.  In that regard it would make sense that the record count would decrease.